docs/CHANGELOG.md

```diff
@@ -71,7 +71,7 @@ The V2 changes were all shipped in [urfave/cli/pull/892](https://github.com/urfa
 ### Added
 
 - Added `NewStringSlice` and `NewIntSlice` for creating their related types
-- Added `Float64SliceFlag` for unmarshaling a list of floats from the user
+- Added `Float64SliceFlag` for unmarshalling a list of floats from the user
```

Comment:
This is correct (per U.S. English –which I suppose is used everywhere in software world)

Reply:
Thank you for your review.
I reverted the fix. 659122249f346ec544384ec23fe0c71f19d0fb10